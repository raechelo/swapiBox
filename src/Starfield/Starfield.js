import React from 'react'
 
import StarfieldAnimation from 'react-starfield-animation'
 
function Starfield() {
  return (
    <StarfieldAnimation
      style={{
        position: 'absolute',
        width: '100%',
        height: '100%'
      }}
    />
  )
}

export default Starfield;