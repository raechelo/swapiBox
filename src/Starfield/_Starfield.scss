body {
  background-color: $space;
}

.Starfield {
  z-index: 1;
}