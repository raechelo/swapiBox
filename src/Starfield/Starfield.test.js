import React from 'react';
import { shallow } from 'enzyme';
import Starfield from './Starfield';

describe('Starfield', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Starfield />
    )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot()
  });

})