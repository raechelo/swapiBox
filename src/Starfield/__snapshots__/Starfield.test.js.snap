// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Starfield should match the snapshot with all the data passed in 1`] = `
<SizeMe(StarfieldAnimation)
  style={
    Object {
      "height": "100%",
      "position": "absolute",
      "width": "100%",
    }
  }
/>
`;
